subsys/net/lib/sockets/Kconfig

```diff
@@ -6,6 +6,7 @@
 menuconfig NET_SOCKETS
```

Comment:
At least for networking commits, there should be no need to mention subsys/ in commit subject. Also there is no sockets subsystem, just networking one.
I suggest just say `net/sockets: Add REQUIRES_MALLOC when enabling sockets`

Reply:
Thanks. Fixed.